### PR TITLE
PLU-70: [TILES-3] Set up dynamodb test container

### PR DESCRIPTION
**Pull request by pregnantboy**
## Dynamodb test container 
- allows testing of dynamodb queries in integration test files
- there's no fast way to truncate the entire table now and i dont see the need to for each test so the setup file hook is left empty
- hiding setup scripts logs by default

### CODE REVIEW

**Review by ogp-weeloong — APPROVED** [review]
LGTM

**Comment by ogp-weeloong** [comment]
I missed some graphql typing problems in the previous review though.

* `Table` type is missing from graphql schema, should rename to `TableMetadata`.
* Need to duplicate `TableColumnMetadata` as a separate input type, as it's being used in `UpdateTableInput`.

**Comment by ogp-weeloong** [comment]
Are you also using `AWS_REGION` or similar in `.env`? Don't forget to update env-example

**Comment by ogp-weeloong** [comment]
Also don't forget to delete the `new dynamoose.Table('table', [TableRow])` in dynamodb.ts

**Comment by pregnantboy** [comment]
> Also don't forget to delete the `new dynamoose.Table('table', [TableRow])` in dynamodb.ts

why?

**Comment by ogp-weeloong** [comment]
> > Also don't forget to delete the `new dynamoose.Table('table', [TableRow])` in dynamodb.ts
> 
> why?

Nvm I derped, didn't see the initialize is false for prod. lets go

File: packages/backend/test/ddb-reset-db-setup.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-console */
+import '../src/config/dynamodb'
+
+import { afterEach } from 'vitest'
+
+afterEach(async () => {
+  // there isnt a good way to truncate all tables
+  // but since there is no the mock tables are re-created each time,
```
Comment (liangyuanruo) [inline packages/backend/test/ddb-reset-db-setup.ts:8]: typo
